src/java.base/share/classes/java/lang/classfile/attribute/StackMapFrameInfo.java

```diff
@@ -61,17 +61,24 @@ public sealed interface StackMapFrameInfo
      */
     List<VerificationTypeInfo> stack();
 
+    /**
+     * {@return the expanded unset fields}
+     */
+    List<Integer> unSetFields();
+
     /**
      * {@return a new stack map frame}
      * @param target the location of the frame
      * @param locals the complete list of frame locals
      * @param stack the complete frame stack
+     * @param unSetFields the complete list of unset fields
      */
     public static StackMapFrameInfo of(Label target,
             List<VerificationTypeInfo> locals,
-            List<VerificationTypeInfo> stack) {
+            List<VerificationTypeInfo> stack,
+            List<Integer> unSetFields) {
```

Comment:
Can keep the original one untouched for compatibility and add a new factory `ofUnsetFields(List<NameAndTypeEntry>)`